group_project_v2/stage/review.py

```diff
@@ -289,14 +370,36 @@ def available_to_current_user(self):
     def is_graded_stage(self):
         return True
 
+    def _get_review_items(self, review_groups, with_caching=False):
+        def do_get_items(group_id):
+            if with_caching:
+                return self._get_review_items_for_group(self.project_api, group_id, self.activity_content_id)
+            else:
+                return self.project_api.get_workgroup_review_items_for_group(group_id, self.activity_content_id)
+
+        return list(itertools.chain.from_iterable(do_get_items(group.id) for group in review_groups))
+
     def review_status(self):
-        group_review_items = []
-        for assess_group in self.review_groups:
-            group_review_items.extend(
-                self.project_api.get_workgroup_review_items_for_group(assess_group["id"], self.content_id)
-            )
+        group_review_items = self._get_review_items(self.review_groups, with_caching=False)
```

Comment:
Please add a comment here or in `_get_review_items` explaining why `with_caching` is False.